modes/corfu/evil-collection-corfu.el

```diff
@@ -89,7 +107,7 @@ This key theme variable may be refactored in the future so use with caution."
   ;; https://github.com/minad/corfu#tab-and-go-completion
   (when (memq 'tab-n-go evil-collection-corfu-key-themes)
     (setq corfu-cycle t
-          corfu-preselect-first nil)
+          corfu-preselect 'prompt)
```

Comment:
This changed a bit ago, so `corfu-preselect-first` didn't actually exist anymore.